pkg/gofr/datasource/pubsub/eventhub/eventhub.go

```diff
@@ -21,7 +20,10 @@ import (
 // metrics are being registered in the container, and we are using the same metrics so we have not re-registered the metrics here.
 // It is different from other datasources.
 
-var errNoMsgReceived = errors.New("no message received")
+var (
+	errNoMsgReceived = errors.New("no message received")
```

Comment:
can we export these error messages as they are returned from the functions? It would make testing implementations easier.